src/styles/components/typeahead/styles.less

```diff
@@ -114,19 +114,3 @@
     }
   }
 }
-
-& when (@typeahead-enabled) and (@layout-screen-large-enabled) {
-
-  @media (min-width: @layout-screen-large-min-width) {
-
-    .form-control-group.filter-input-text-group {
-
-      .typeahead {
-
-        .dropdown-menu {
-          margin-left: calc(-1 * (@icon-mini-width + @form-control-border-width) ~' - ' (-2 * @form-control-padding-left-screen-large));
-        }
-      }
-    }
-  }
-}
```

Comment:
Because the new variable doesn't have corresponding screen sizes (it's a `rem` unit), we don't need to change our declaration at the `large` breakpoint.